gnovm/stdlibs/math/all_test.gno

```diff
@@ -2,31 +2,593 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
+// XXX:
+// When porting over to Gno, this file has been changed in the name
+// of its variables and in avoiding using a dot import (which are not allowed
+// in Gno.) In most cases, variable names have a simple underscore to prefix
+// them and not come into conflict with other identifiers in the `math` package.
+// Other changes involve removing code that pertained to comparing the "Go"
+// implementation and the "assembly" implementation (in our case, we currently
+// only have the Gno implementation).
+
 package math
 
 import (
+	"fmt"
 	"testing"
 )
 
+var _vf = []float64{
```

Comment:
Why all the underscore prefixes?

Reply:
See the comment at the beginning of the file

Go uses a dot import of math in order to have variables in the test files, even if their identifiers are duplicate with other identifiers in the math package

We cannot have this as we don't support dot imports in the first place

Reply:
Ah I see. Can the package name be `math_test` like the original go file and then just import math as usual? I don't think there would be any identical identifiers if they are unexported and in separate packages.

Reply:
yes, but then we would need to prefix all calls to functions with `math.`, and I tend to think that's undesirable and will cause more diff from the original. wdyt?

Reply:
Hmm things like this aren't super important, but I still have an opinion on them -- I think it's preferable for test files to be in `_test` suffixed packages in order to limit tests' visibility; this ensures that the behavior is tested and not the implementation as no code that imports the package can do anything that can't be invoked using the exported API. The dot import is sloppy in my opinion because it leads to less readable code -- where is this function/type defined? Also variables with an underscore prefix in go is not usual and discouraged.

Hopefully that convinced you. If it didn't that's also fine; it's unlikely a lot of people will be reading the implementation.